webapp/components/interop-dashboard.js

```diff
@@ -344,10 +345,19 @@ class InteropDashboard extends PolymerElement {
                       <th></th>
                     </template>
                   </tr>
+                  <tr>
+                  <template is="dom-if" if="[[showSortToggle(itemsIndex)]]">
+                    <td><paper-icon-button class="sort-button" id="col-0" on-click="handleSortClick" src="[[getSortIcon(0, sortToggled)]]"></paper-icon-button></td>
+                    <td><paper-icon-button class="sort-button" id="col-1" on-click="handleSortClick" src="[[getSortIcon(1, sortToggled)]]"></paper-icon-button></td>
+                    <td><paper-icon-button class="sort-button" id="col-2" on-click="handleSortClick" src="[[getSortIcon(2, sortToggled)]]"></paper-icon-button></td>
+                    <td><paper-icon-button class="sort-button" id="col-3" on-click="handleSortClick" src="[[getSortIcon(3, sortToggled)]]"></paper-icon-button></td>
+                    <td><paper-icon-button class="sort-button" id="col-4" on-click="handleSortClick" src="[[getSortIcon(4, sortToggled)]]"></paper-icon-button></td>
+                  </template>
+                </tr>
```

Comment:
You can move the `<tr>` tag into the template so that an empty row will not be created if it's not needed.